translations/hu/get5.phrases.txt

```diff
@@ -30,7 +30,7 @@
     }
     "TypeUnreadyIfNotReady"
     {
-        "hu"            "Írd be, hogy {1} ha nem állsz még készen."
+        "hu"            "Használd a {1}, hogy ha nem állsz még készen."
```

Comment:
`Használd a {1} parancsot, hogy ha nem állsz még készen.`

Reply:
Elfogadva a javaslat. The suggestion is accepted